man/Add.Rd

```diff
@@ -0,0 +1,21 @@
+% Generated by roxygen2: do not edit by hand
```

Comment:
I'm confused why this file is called "Add.Rd" but I have a feeling it's left over from running the roxygen before this function was renamed? I don't know. Either way, we probably wanna `git rm` it and run `devtools::document()` again to refresh it. If it comes up weirdly named again, there may be an issue I'm not spotting in the roxygen code.